awx/ui/client/src/inventories-hosts/hosts/hosts.partial.html

```diff
@@ -64,17 +65,22 @@
                                 </div>
                             </div>
                             <div class="d-flex h-100">
-                                <div class="List-tableCell name-column InventoryManage-breakWord col-lg-4">
+                                <div class="List-tableCell name-column InventoryManage-breakWord col-lg-3">
                                     <div class="host-name">
                                         <a ui-sref="hosts.edit({host_id:host.id})">{{host.name }}</a>
                                     </div>
                                 </div>
-                                <div class="List-tableCell inventory_name-column col-lg-4 elllipsis">
+                                <div class="List-tableCell name-column InventoryManage-breakWord col-lg-3">
```

Comment:
@jlariza I don't think we want the description to be a link - we don't have that pattern anywhere else in the application.  I think you'll want this code to look like:

```
                                <div class="List-tableCell name-column InventoryManage-breakWord col-lg-3">
                                    {{host.description }}
                                </div>
```